desktop/release-notes.md

```diff
@@ -24,12 +24,95 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.14
+
+2022-11-10
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Upgrades
+
+- [Docker Engine v20.10.21](https://docs.docker.com/engine/release-notes/#201021),
+  which contain mitigations against a Git vulnerability, tracked in [CVE-2022-39253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-39253){:target="\_blank" rel="noopener"},
+  and updated handling of `image:tag@digest` image references.
+- [Docker Compose v2.12.2](https://github.com/docker/compose/releases/tag/v2.12.2)
+- [Containerd v1.6.9](https://github.com/containerd/containerd/releases/tag/v1.6.9)
+- [Go 1.19.3](https://github.com/golang/go/releases/tag/go1.19.3)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Docker Desktop now requires an internal network subnet of size /24. If you were previously using a /28, it will be automatically expanded to /24. If you experience networking issues, check to see if you have a clash between the Docker subnet and your infrastructure. Fixes [docker/for-win#13025](https://github.com/docker/for-win/issues/13025)
+- Fixes an issue that prevents users from creating Dev Environments when the Git URL has upper case characters
+- Fix error `vpnkit.exe is not running` reported in diagnostics
+- Revert qemu to 6.2.0 to fix errors like `PR_SET_CHILD_SUBREAPER is unavailable` when running emulated amd64 code.
+- Enabled [contextIsolation](https://www.electronjs.org/docs/latest/tutorial/context-isolation) and [sandbox](https://www.electronjs.org/docs/latest/tutorial/sandbox) mode inside Extensions. Now Extensions run in a separate context and this limits the harm that malicious code can cause by limiting access to most system resources.
+- Included `unpigz` to allow parallel decompression of pulled images.
+- Fixed issues related to performing action on selected containers (https://github.com/docker/for-win/issues/13005)
+- Adds the possibility of displaying timestamps for the container / project view.
+- Fix a possible segfault when interrupting `docker pull` with Control+C
+- Increase the default DHCP lease time to avoid the VM's network glitching and dropping connections every 2 hours
+- Remove infinite spinner on containers list (https://github.com/docker/for-mac/issues/6486)
+- Fixed bug showing incorrect values on used space in Settings
+- Fixed kubernetes not starting with the containerd integration
+- Fixed `kind` not starting with the containerd integration
+- Fixed dev environments not working with the containerd integration
+- Implemented `docker diff` in the containerd integration
+- Implemented `docker run —-platform` in the containerd integration
+- Fixed insecure registries not working with the containerd integration
+
+#### For Mac
+
+- Promoted Virtualization framework to default hypervisor for macOS >= 12.5.
```

Comment:
This is a big change, I would move it up to the New section.

Reply:
done